docs/guide/user_management/user_management.md

```diff
@@ -17,6 +17,41 @@ You can [customize it according to your needs](#customize-login-form).
 The validity of the password recovery token can be set using the `ezplatform.system.<siteaccess>.security.token_interval_spec` parameter.
 By default it is set to `PT1H` (one hour).
 
+### Add forget password option in site front
+
+You can add a forgot password option to the site front to allow users of a specific site request a password change.
+
+Edit `congfig/packages/ezplatform.yaml`. The templates are defined with the following configuration:
+
+```yaml
+ezplatform:
+    system:
+        <siteaccess>:
+            user_forgot_password:
+                templates:
+                    form: <path_to_template>
+                    mail: <path_to_template>
+```
+
+The [templates](https://github.com/ezsystems/ezplatform-admin-ui/blob/master/src/bundle/Resources/views/themes/admin/account/forgot_password/) for forgot password form and email are located in `/src/bundle/Resources/views/themes/admin/account/` in `ezsystems/ezplatform_admin_ui`.
+
+You can also modify the following form templates:
+
+**Forgot password forms:**
+
+```yaml
+user_change_password.templates.success
+user_forgot_password.templates.form
+user_forgot_password.templates.mail
+user_forgot_password_success.templates.form
+user_forgot_password_login.templates.form
+user_reset_password.templates.form
```

Comment:
These are actually mentioned already in https://doc.ibexa.co/en/master/guide/user_management/user_management/#other-user-management-templates

Reply:
They also gonna have proper semantic configuration once https://github.com/ezsystems/ezplatform-user/pull/88 is merged. On the other hand in this state those params names are not complete here. I would rather put full semantic configuration somewhere.